requirements-test.txt

```diff
@@ -1,3 +1,4 @@
 click==7.0
 ruamel.yaml==0.15.89
 flake8==3.7.7
+mistune==0.8.4
```

Comment:
Since `mistune` version `0.8.4` does not support terminal rendering (only our fork does), we will need to either make sure that my patch is merged into a release version of mistune, or we should bundle our custom version with this package.

Reply:
Agreed, what do you think is the best way to include the patch version? I noticed that coincidently the mistune author is also started working on mistune very recently and is planning on a 2.0 version. Did you have any contact with the author to merge or do you prefer we bundle the fork version in signac?

Reply:
The patch is already based on the v2 code. If we think that this is sufficient, then I can try to create a pull request for mistune. If it gets accepted then we can just add this as a dependency, otherwise I'd be ok with vendoring the fork.

Reply:
@csadorf  I think the mistune branch you made serve our need well, and I agree if you could finalize and merge it to mistune, then us adding mistune 2.0 as dependency would be the easiest solution.

Reply:
I've created a pull request: https://github.com/lepture/mistune/pull/208